deps/libmediasoupclient/src/ortc.cpp

```diff
@@ -103,7 +103,8 @@ namespace mediasoupclient
 				MSC_THROW_TYPE_ERROR("missing codec.mimeType");
 
 			std::smatch mimeTypeMatch;
-			std::regex_match(mimeTypeIt->get<std::string>(), mimeTypeMatch, MimeTypeRegex);
```

Comment:
Idem, these are changes needed because of C++14. Not needed if using libwebrtc m79 branch.